bazel/external/quiche.patch

```diff
@@ -0,0 +1,44 @@
+--- http2/adapter/callback_visitor.cc
++++ http2/adapter/callback_visitor.cc
+@@ -112,6 +112,6 @@ bool CallbackVisitor::OnFrameHeader(Http
+ void CallbackVisitor::OnSettingsStart() {}
+
+ void CallbackVisitor::OnSetting(Http2Setting setting) {
+-  settings_.push_back({.settings_id = setting.id, .value = setting.value});
```

Comment:
nit: can we fix these in upstream instead of patching in Envoy?

Reply:
Yes, fixes are in flight. But quiche is updated weekly, and I wanted to send this out for review without having to wait for next week's quiche update.

Reply:
Sweet. Thanks for doing this!